tests/iscsi/iscsi_client.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+use mm_network;
+use lockapi;
+
+sub run() {
+    my $self = shift;
+
+    x11_start_program("xterm -geometry 160x45+5+5");
+    become_root;
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/graphs/contributors -> you nr. 5, me nr. 12 :-)